x-pack/solutions/security/plugins/security_solution/public/asset_inventory/components/search_bar.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { css } from '@emotion/react';
+import { type EuiThemeComputed, useEuiTheme } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { Filter } from '@kbn/es-query';
+import { useKibana } from '../../common/lib/kibana';
+import { FiltersGlobal } from '../../common/components/filters_global/filters_global';
+import { useDataViewContext } from '../hooks/data_view_context';
+import type { AssetsBaseURLQuery } from '../hooks/use_asset_inventory_data_table';
+
+type SearchBarQueryProps = Pick<AssetsBaseURLQuery, 'query' | 'filters'>;
+
+interface AssetInventorySearchBarProps {
+  setQuery(v: Partial<SearchBarQueryProps>): void;
+  loading: boolean;
+  placeholder?: string;
+  query: SearchBarQueryProps;
+}
+
+export const AssetInventorySearchBar = ({
+  loading,
+  query,
+  setQuery,
+  placeholder = i18n.translate(
+    'xpack.securitySolution.assetInventory.searchBar.searchPlaceholder',
+    {
+      defaultMessage: 'Search findings (eg. rule.section : "API Server" )',
+    }
+  ),
```

Comment:
Let's change the placeholder text as this was related to the findings page

```suggestion
  placeholder = i18n.translate(
    'xpack.securitySolution.assetInventory.searchBar.searchPlaceholder',
    {
      defaultMessage: 'Filter your data using KQL syntax',
    }
  ),
```